[2022-01-29 10:00:47,227] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-02-02T06:00:00+00:00 [queued]>
[2022-01-29 10:00:47,243] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDag.ingest scheduled__2021-02-02T06:00:00+00:00 [queued]>
[2022-01-29 10:00:47,243] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 10:00:47,243] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-01-29 10:00:47,243] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-29 10:00:47,261] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2021-02-02 06:00:00+00:00
[2022-01-29 10:00:47,267] {standard_task_runner.py:52} INFO - Started process 2889 to run task
[2022-01-29 10:00:47,272] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest', 'scheduled__2021-02-02T06:00:00+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmp2u0wwryk', '--error-file', '/tmp/tmpacn0u_9e']
[2022-01-29 10:00:47,273] {standard_task_runner.py:77} INFO - Job 87: Subtask ingest
[2022-01-29 10:00:47,363] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDag.ingest scheduled__2021-02-02T06:00:00+00:00 [running]> on host 68102482cc2e
[2022-01-29 10:00:47,439] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2021-02-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-02T06:00:00+00:00
[2022-01-29 10:00:47,440] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-29 10:00:47,441] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-01-29 10:00:47,452] {subprocess.py:85} INFO - Output:
[2022-01-29 10:00:47,457] {subprocess.py:89} INFO - ***-worker.pid
[2022-01-29 10:00:47,457] {subprocess.py:89} INFO - ***.cfg
[2022-01-29 10:00:47,457] {subprocess.py:89} INFO - dags
[2022-01-29 10:00:47,457] {subprocess.py:89} INFO - logs
[2022-01-29 10:00:47,457] {subprocess.py:89} INFO - output.csv
[2022-01-29 10:00:47,457] {subprocess.py:89} INFO - output_2021-02.csv
[2022-01-29 10:00:47,457] {subprocess.py:89} INFO - plugins
[2022-01-29 10:00:47,457] {subprocess.py:89} INFO - requirements.txt
[2022-01-29 10:00:47,457] {subprocess.py:89} INFO - webserver_config.py
[2022-01-29 10:00:47,458] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-29 10:00:47,486] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDag, task_id=ingest, execution_date=20210202T060000, start_date=20220129T100047, end_date=20220129T100047
[2022-01-29 10:00:47,524] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-29 10:00:47,561] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
